packages/components/src/font-size-picker/font-size-picker-toggle-group.tsx

```diff
@@ -29,7 +46,12 @@ const FontSizePickerToggleGroup = ( props: FontSizePickerToggleGroupProps ) => {
 				<ToggleGroupControlOption
 					key={ fontSize.slug }
 					value={ fontSize.size }
-					label={ T_SHIRT_ABBREVIATIONS[ index ] }
+					label={
+						fontSize.shortName &&
+						hasValidShortName( fontSize.shortName )
```

Comment:
```suggestion
						hasValidShortName( fontSize?.shortName )
```

Suggestion: The `hasValidShortName` will return false if short name is undefined.

Reply:
@Mamaduka the reason I added back in the duplicate check is that TypeScript is complaining about it for whatever reason... See https://github.com/WordPress/gutenberg/actions/runs/3706287491/jobs/6281315087

Reply:
I just noticed it. Sorry about that. Let's revert for now, and maybe more TS-savvy people can help us here.

Reply:
Reverted in dc9fdd4403757019607d63c45c5a6b681e9e59fa

Reply:
It should work to remove this so long as `hasValidShortName`'s argument type is `string | undefined`.

Reply:
Yeah for whatever reason that doesn't work for me. As per your later comment, I now removed the `| undefined` type from the function and kept the duplicate check inline.